classes.py

```diff
@@ -13,6 +14,11 @@ class Page(TypedDict):
     url: str
     
 
+    def eq(self, other):
```

Comment:
TypedDicts can only contain type annotations. This can't be here, and I don't think it needs to be anyway.